mesos_slave/tests/common.py

```diff
@@ -14,6 +14,7 @@
 FIXTURE_DIR = os.path.join(HERE, 'fixtures')
 CHECK_NAME = 'mesos_slave'
 URL = 'http://{}:5051'.format(get_docker_hostname())
+MESOS_SLAVE_VERSION = '0.22.0'
```

Comment:
Instead, do https://github.com/DataDog/integrations-core/pull/4818/files#diff-3fe965a859ad39106c20b66c72cdba47 and make use of `tox`

Reply:
actually it was hard coded there because the tests pull in the fixture data which has a fixed version (0.22.0). I'm checking to see if there's a better way around this